app/actors/Actors.scala

```diff
@@ -1,47 +1,46 @@
 package actors
 
+import javax.inject._
+
+import akka.actor.{ActorRef, ActorSystem}
+import com.google.inject.AbstractModule
 import play.api._
-import play.api.libs.concurrent.Akka
+import play.api.libs.concurrent.AkkaGuiceSupport
 import backend._
 import akka.cluster.Cluster
 import java.net.URL
 
 /**
- * Lookup for actors used by the web front end.
+ * Guice module that provides actors.
+ *
+ * Registered in application.conf.
  */
-object Actors {
-
-  private def actors(implicit app: Application) = app.plugin[Actors]
-    .getOrElse(sys.error("Actors plugin not registered"))
-
-  /**
-   * Get the region manager client.
-   */
-  def regionManagerClient(implicit app: Application) = actors.regionManagerClient
+class Actors extends AbstractModule with AkkaGuiceSupport {
+
+  def configure() = {
+    // Bind the region manager client
+    bindActor[RegionManagerClient]("regionManagerClient")
```

Comment:
It's quite easy to bind an `ActorSelection`, though it has to be bound explicitly for each actor, there's no way in Guice to dynamically provide named bindings on demand, and we don't currently provide any facilities to assist with it. Binding a child `ActorRef` probably wouldn't make sense, since children are managed by their parent, not Guice, but we do provide facilities to assist with injecting children via Guice assisted inject factories - this is really useful for example at the bottom of your hierarchy you need a new injected dependency, with Akka generally this would require injecting the complement all the down the hierarchy, but using these facilities, you just need to add the dependency to the one actors constructor and you're done.